<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRxDisplay</class>
 <widget class="QFrame" name="CRxDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1" columnminimumwidth="150,0,0">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="TabWidget" name="RxHeader">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>546</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Clear">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>13</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="RecordBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="recordActivate">
          <property name="text">
           <string>RecordQso</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="TriggerBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="title">
        <string>Trigger</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLineEdit" name="TriggerText">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="text">
           <string>CQ CQ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="triggerActivate">
          <property name="text">
           <string>Activate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="FrequencySelect" name="RxFreq">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="title">
        <string>Rx Frequency / AFC</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="CSquelch" name="Squelch">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Squelch</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrequencySelect</class>
   <extends>QGroupBox</extends>
   <header>frequencyselect.h</header>
   <container>1</container>
   <slots>
    <signal>FrequencyChanged(double)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>CSquelch</class>
   <extends>QGroupBox</extends>
   <header>csquelch.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>RxFreq</sender>
   <signal>FrequencyChanged(double)</signal>
   <receiver>CRxDisplay</receiver>
   <slot>setRxFrequency(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RxHeader</sender>
   <signal>currentChanged(int)</signal>
   <receiver>CRxDisplay</receiver>
   <slot>changeActiveRxWindow(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Clear</sender>
   <signal>clicked()</signal>
   <receiver>CRxDisplay</receiver>
   <slot>clearRxWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>833</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triggerActivate</sender>
   <signal>clicked()</signal>
   <receiver>CRxDisplay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TriggerText</sender>
   <signal>returnPressed()</signal>
   <receiver>CRxDisplay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordActivate</sender>
   <signal>toggled(bool)</signal>
   <receiver>CRxDisplay</receiver>
   <slot>record(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setRxFrequency(double)</slot>
  <slot>changeActiveRxWindow(int)</slot>
  <slot>clearRxWindow()</slot>
  <slot>trigger()</slot>
  <slot>record(bool)</slot>
 </slots>
</ui>
